README.md

```diff
@@ -1,10 +1,17 @@
+# ⚠️ This repository is deprecated.
+
+kube2pulumi has been deprecated and is no longer maintained. Please note that this repository is no longer actively developed or supported.
```

Comment:
Let's soften this a bit for now to give customer's a chance to try the new thing with less pressure to abandon kube2pulumi. How about:

```suggestion
# 📌 Please try the new Kubernetes converter plugin! 📌

```